libraries/APM_OBC/APM_OBC.cpp

```diff
@@ -117,6 +117,30 @@ const AP_Param::GroupInfo APM_OBC::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("MAX_COM_LOSS", 14, APM_OBC, _max_comms_loss, 0),
 
+    // @Param: GEOFENCE
+    // @DisplayName: Enable geofence Advanced Failsafe
+    // @Description: This enables the geofence part of the AFS. Will only be in effect if AFS_ENABLE is also 1
+    // @User: Advanced
+    AP_GROUPINFO("GEOFENCE",     15, APM_OBC, _enable_geofence_fs, 1),
```

Comment:
My intention was to leave all of the default settings the same as when I found them, but I'm happy to make the default value disabled (especially since it will be disabled for my purposes) if you think that's better practice. Do you think that I should also make the dual loss termination and RC failsafes disabled by default as well then?